docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-product-option-feature-integration.md

```diff
@@ -0,0 +1,465 @@
+     */
+    protected function getProductOptionListTableQueryCriteriaExpanderPlugins(): array
+    {
+        return [
+            new MerchantProductOptionListTableQueryCriteriaExpanderPlugin(),
+        ];
+    }
+
+    /**
+     * @return \Spryker\Zed\ProductOptionExtension\Dependency\Plugin\ProductOptionGroupExpanderPluginInterface[]
+     */
+    protected function getProductOptionGroupExpanderPlugins(): array
+    {
+        return [
+            new MerchantProductOptionGroupExpanderPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/ProductOptionStorage/ProductOptionStorageDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\ProductOptionStorage;
+
+use Spryker\Zed\MerchantProductOptionStorage\Communication\Plugin\ProductOptionStorage\MerchantProductOptionCollectionFilterPlugin;
+use Spryker\Zed\ProductOptionStorage\ProductOptionStorageDependencyProvider as SprykerProductOptionStorageDependencyProvider;
+
+class ProductOptionStorageDependencyProvider extends SprykerProductOptionStorageDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\ProductOptionStorageExtension\Dependency\Plugin\ProductOptionCollectionFilterPluginInterface[]
+     */
+    protected function getProductOptionCollectionFilterPlugins(): array
+    {
+        return [
+            new MerchantProductOptionCollectionFilterPlugin(),
+        ];
+    }
+}
+```
+
+**src/Pyz/Zed/Publisher/PublisherDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Publisher;
+
+use Spryker\Zed\MerchantProductOptionStorage\Communication\Plugin\Publisher\MerchantProductOption\MerchantProductOptionGroupWritePublisherPlugin;
+use Spryker\Zed\Publisher\PublisherDependencyProvider as SprykerPublisherDependencyProvider;
+
+class PublisherDependencyProvider extends SprykerPublisherDependencyProvider
+{
+    /**
+     * @return array
+     */
+    protected function getPublisherPlugins(): array
+    {
+        return array_merge(
+            // ...
+            $this->getMerchantProductOptionStoragePlugins()
+        );
+    }
+    
+    /**
+     * @return \Spryker\Zed\PublisherExtension\Dependency\Plugin\PublisherPluginInterface[]
+     */
+    protected function getMerchantProductOptionStoragePlugins(): array
+    {
+        return [
+            new MerchantProductOptionGroupWritePublisherPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure merchants can create product option groups and values in the merchant portal.
+Make sure that merchant product option information is shown on product details page, when it is approved and active.
+Make sure that merchant product option information is displayed in the cart, checkout and in the user account.
+Make sure that merchant product options are part of a marketplace/merchant order and all totals are calculated correctly.
+
+{% endinfo_block %}
+
+
+### 5) Add Yves translations
```

Comment:
Invalid section name, please see https://spryker.atlassian.net/wiki/spaces/DOCS/pages/478118714/Feature+integration+guides for the allowed list. 

(For the reference https://spryker.atlassian.net/wiki/spaces/DOCS/pages/1072235833/Configurable+Bundle+Feature+Integration+-+ongoing can be used)